### ISSUE TITLE
Electron 4.x.y macOS builds crash

### ISSUE THREAD

**Issue by VishwasShashidhar** [electron-userland/electron-builder#3452]
* **Version**: 20.32.1
* **Target**: macOS mojave / Electron 4.0.0-beta.5

When I'm trying to build on macOS mojave with Electron versions > 4.x.y, the app is built, but the app crashes upon launch. Please find below the crash report. It looks like something to do with symlinks.

```
Process:               myapp [20971]
Path:                  /Applications/myapp.app/Contents/MacOS/myapp
Identifier:            com.myapp.electron-desktop
Version:               3.4.0 (3.4.0)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           myapp [20971]
User ID:               501

Date/Time:             2018-11-06 16:00:13.112 +0530
OS Version:            Mac OS X 10.14 (18A391)
Report Version:        12
Bridge OS Version:     3.0 (14Y667)
Anonymous UUID:        8D08FD02-58BB-5736-864C-7BB7FF0C5C01

Sleep/Wake UUID:       D39E9E49-4933-4E89-A61A-FD0987F5298E

Time Awake Since Boot: 51000 seconds
Time Since Wake:       2900 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/Electron Framework.framework/Electron Framework
  Referenced from: /Applications/myapp.app/Contents/MacOS/myapp
  Reason: no suitable image found.  Did find:
	/Applications/myapp.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: stat() failed with errno=13

Binary Images:
       0x104c09000 -        0x104c09fff +com.myapp.electron-desktop (3.4.0 - 3.4.0) <A66AAB6A-8A4B-30C9-A708-9C2A009B2C83> /Applications/myapp.app/Contents/MacOS/myapp
       0x10e91c000 -        0x10e99a6a7  dyld (625.13) <D6387150-2FB8-3066-868D-72E1B1C43982> /usr/lib/dyld
    0x7fff5047c000 -     0x7fff5047cfff  com.apple.Cocoa (6.11 - 23) <A520D2A3-032B-3C06-81E1-EFDD97C47ACE> /System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa
    0x7fff5301d000 -     0x7fff533ebfff  com.apple.Foundation (6.9 - 1555.10) <0CCAF9AF-F7AF-3174-98D4-6C188B1079CC> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
    0x7fff53710000 -     0x7fff537a1fff  com.apple.framework.IOKit (2.0.2 - 1483.201.1) <DA4ED91F-2CC9-3CFD-9200-9D5D31EEE4F3> /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
    0x7fff5c6fc000 -     0x7fff5ca22fff  com.apple.security (7.0 - 58286.200.222) <AEB638D9-BFE9-319E-A70B-557ACF41117A> /System/Library/Frameworks/Security.framework/Versions/A/Security
    0x7fff5cea5000 -     0x7fff5cf15ff3  com.apple.SystemConfiguration (1.17 - 1.17) <EED5BC9B-CE8D-343C-BCDC-04A79F704828> /System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration

```

**Comment by VishwasShashidhar** [comment]
looks similar to #3435 except for Electron 4.x.y beta versions being used

**Comment by VishwasShashidhar** [comment]
@develar I've created a new ticket as per your comment in #3435

**Comment by develar** [comment]
I cannot reproduce with 4.0.0-beta7 and electron-builder 20.33.2. Upcoming 20.33.0 will contain some improvements to unzip, but I don't see any issues. Please try when 20.33.0 will be released.

**Comment by VishwasShashidhar** [comment]
@develar I can reproduce this, it only happens for macOS target "pkg", works cleanly for "dmg" and "zip". Can you please look into it? Also, do you want me to create a new ticket for this or can you reopen this ticket?

**Comment by VishwasShashidhar** [comment]
Also, I'm using the latest & greatest version i.e. `20.34.0`

**Comment by develar** [comment]
@VishwasShashidhar Fixed in upcoming 20.37.0
